media-sound/scream/scream-3.7.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Plays sound received from network or from a QEMU Windows VM"
+HOMEPAGE="https://github.com/duncanthrax/scream"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/duncanthrax/scream.git"
+else
+	SRC_URI="https://github.com/duncanthrax/scream/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+S="${WORKDIR}/${P}/Receivers/unix"
```

Comment:
doesn't this need to go into the "else" block above?

Reply:
No, why?
This is true for both cases.